Physics/HelicityFormalism/HelicityKinematics.hpp

```diff
@@ -75,14 +75,6 @@ class HelicityKinematics : public ComPWA::Kinematics {
       const std::vector<pid> &finalState,
       const ComPWA::FourMomentum &cmsP4 = ComPWA::FourMomentum(0, 0, 0, 0));
 
-  /// Delete copy constructor. For each Kinematics in the analysis only
-  /// one instance should exist since Kinematics does the bookkeeping which
-  /// SubSystems variables are needs to be calculated. That instance can then be
-  /// passed as (smart) pointer.
-  /// Note1: Not sure if we also should delete the move constructor.
-  /// Note2: We have to delete the copy constructor in Base and Derived classes.
-  HelicityKinematics(const HelicityKinematics &that) = delete;
```

Comment:
Isn't that statement still true? Why should we not delete the copy constructor here?

Reply:
Hmm, yeah that is true. I have to double check why I removed this....